src/material/chips/chip-listbox.ts

```diff
@@ -158,6 +158,18 @@ export class MatChipListbox
   }
   protected _required: boolean = false;
 
+  /** Whether checkmark indicator for single-selection options is hidden. */
+  @Input()
+  get hideSingleSelectionIndicator(): boolean {
+    return this._hideSingleSelectionIndicator;
+  }
+  set hideSingleSelectionIndicator(value: BooleanInput) {
+    this._hideSingleSelectionIndicator = coerceBooleanProperty(value);
+    this._syncListboxProperties();
+  }
+  protected _hideSingleSelectionIndicator: boolean =
```

Comment:
This one should be private since we have access to it through the getter already.